### PR TITLE
Use flexbox in speakers layer

### PR DESCRIPTION
**Pull request by cameronblandford**

### CODE REVIEW

File: app/styles/components/contributor-list.scss [hunk 1]
```diff
@@ -17,6 +17,12 @@
       }
       color: #222;
     }
+    width: 30%;
+
+    @media (max-width: 599px ) { width: calc(50% - 30px);      } // XS
+    @media (min-width: 600px ) { width: calc(33.33333333% - 30px);       } // SM
+    @media (min-width: 980px ) { width: calc(25% - 30px);       } // LG
+    @media (min-width: 1234px) { width: calc(16.6666666% - 30px);  } // MD
```
Comment (birdbrained) [inline app/styles/components/contributor-list.scss]: woah you can do -media queries- inside the selector in sass?

File: app/styles/components/contributor-list.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.meeting-keynote-speakers-container {
+.contributor-list-container {
 
   .speaker-pic-thumbnail {
```
Comment (birdbrained) [inline app/styles/components/contributor-list.scss]: do we want to use speaker or contributor?

File: app/components/contributor-list/template.hbs [hunk 3]
```diff
@@ -8,10 +8,10 @@
                 </h2>
             </div>
         {{/if}}
-        <div class="row text-center">
+        <div class="flex-container">
```
Comment (birdbrained) [inline app/components/contributor-list/template.hbs]: maybe speaker-list or contributor-list?

Comment (cameronblandford) [reply app/components/contributor-list/template.hbs]: good point!

File: app/styles/components/contributor-list.scss (hunk 2 above)
Comment (cameronblandford) [reply app/styles/components/contributor-list.scss]: contributor - good point again!

**Pull request merged**